### ISSUE TITLE
Support ipv6 in `FilterOptions`

### ISSUE THREAD

**Issue by MohamedElmdary** [threefoldtech/tfgrid-sdk-ts#155]
Currently, **public_ip_ipv6** is not included in `FilterOptions`

**Comment by AhmedHanafy725** [comment]
require: https://github.com/threefoldtech/tfgrid-sdk-go/issues/926

**Comment by zaelgohary** [comment]
Work Completed:

Added ipv6 to Node Finder filters.

**Comment by zaelgohary** [comment]
Blocked on https://github.com/threefoldtech/tfgrid-sdk-go/issues/1040

**Comment by zaelgohary** [comment]
Work completed:

Included node_has_ipv6 in farm filters in grid client, proxy & playground.

**Comment by A-Harby** [comment]
Devnet, 44f15ae.

The IPV6 filter has been added to the node finder, solution, and TS client filters.
Also, the IPV6 endpoint was added to the gridproxy.

Also now, the IPV4 is called free_ips, while the IPV6 is the IPV6.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/109473873/dc139c91-54d5-4c3b-96b9-3e13eebd0ba0)
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/109473873/61cfba54-93f2-4e7e-99c6-46225f643e3f)
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/109473873/016e3dd5-8a4a-4384-bb84-ac76cc5cda89)

[TC2714 - Filter IPV6 nodes](https://app.testlodge.com/a/26076/projects/40893/suites/234374?expand_section=426661#case_4295008)

I opened a separate issue to provide IPV6 as an option in network options beside IPV4: https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2930.
